data/blank_mod/common/ideas/air_spirits.txt

```diff
@@ -0,0 +1,291 @@
+			ledger = air
+			visible = { has_tech = formation_flying }
+			available = { has_air_academy = yes }
+			research_bonus = {
+				cas_bomber = 0.05
+			}
+			modifier = {
+				small_plane_cas_airframe_design_cost_factor = -0.75
+				cv_small_plane_cas_airframe_design_cost_factor = -0.75
+				#CAS_equipment_design_cost_factor = -0.75
+				#cv_CAS_equipment_design_cost_factor = -0.75
+			}
+			ai_will_do = {
+				factor = 1
+				modifier = {
+					factor = 0
+					NOT = { has_dlc = "No Step Back" }
+				}
+			}
+		}
+		material_destruction_spirit = {
+			ledger = air
+			visible = { has_tech = force_rotation }
+			available = { has_air_academy = yes }
+			research_bonus = {
+				tactical_bomber = 0.05
+			}
+			modifier = {
+				medium_plane_airframe_design_cost_factor = -0.75
+				#tac_bomber_equipment_design_cost_factor = -0.75
+				#jet_tac_bomber_equipment_design_cost_factor = -0.75
+			}
+			ai_will_do = {
+				factor = 1
+				modifier = {
+					factor = 0
+					NOT = { has_dlc = "No Step Back" }
+				}
+			}
+		}
+		industry_liasons_spirit = {
+			ledger = air
+			available = { has_air_academy = yes }
+			research_bonus = {
+				light_air = 0.15
+				medium_air = 0.15
+				heavy_air = 0.15
+				naval_air = 0.15
+			}
+			ai_will_do = {
+				factor = 1
+				modifier = {
+					factor = 0
+					NOT = { has_dlc = "No Step Back" }
+				}
+			}
+		}
+		
+		effective_training_programs_spirit = {
+			ledger = air
+			available = { has_air_academy = yes }
+			modifier = {
+				air_training_xp_gain_factor = 0.25
+			}
+			ai_will_do = {
+				factor = 1
+				modifier = {
+					factor = 0
+					NOT = { has_dlc = "No Step Back" }
+				}
+			}
+		}
+		branch_independence_spirit = {
+			ledger = air
+			available = { has_air_academy = yes }
+			modifier = {
+				experience_gain_air = 0.1
+				airforce_intel_to_others = -5
+			}
+			ai_will_do = {
+				factor = 1
+				modifier = {
+					factor = 0
+					NOT = { has_dlc = "No Step Back" }
+				}
+			}
+		}
+		air_crew_surveys_spirit = {
+			ledger = air
+			available = { has_air_academy = yes }
+			modifier = {
+				air_doctrine_cost_factor = -0.15
+				air_accidents_factor = -0.25
+			}
+			ai_will_do = {
+				factor = 1
+				modifier = {
+					factor = 0
+					NOT = { has_dlc = "No Step Back" }
+				}
+			}
+		}
+	}
+
+	air_force_command_spirit = {
+		battlefield_air_interdiction_spirit = {
+			ledger = air
+			available = { has_air_force_command = yes }
+			modifier = {
+				ground_attack_factor = 0.05
+				air_escort_efficiency = 0.05
+			}
+			ai_will_do = {
+				factor = 1
+				modifier = {
+					factor = 0
+					NOT = { has_dlc = "No Step Back" }
+				}
+			}
+		}
+		cag_night_fighting_spirit = {
+			ledger = air
+			visible = { tag = ENG }
+			modifier = {
+				air_carrier_night_penalty_reduction_factor = 0.33
+				carrier_night_traffic = 0.5
+			}
+			ai_will_do = {
+				factor = 1
+				modifier = {
+					factor = 0
+					NOT = { has_dlc = "No Step Back" }
+				}
+			}
+		}
```

Comment:
This one is country-specific, please remove.

Reply:
Done.